cmd/heal.go

```diff
@@ -0,0 +1,359 @@
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/minio/minio-go/v6"
+	"github.com/minio/minio-go/v6/pkg/encrypt"
+	"github.com/minio/minio/pkg/sync/errgroup"
+	"github.com/minio/radio/cmd/logger"
+	"github.com/vmihailenco/msgpack"
```

Comment:
No, we shouldn't use this one @poornas https://github.com/tinylib/msgp look at how to do it in the versioning PR, lets discuss tomorrow on how to do an optimized write for this instead of new line